man/systemd-pcrphase.service.xml

```diff
@@ -21,10 +21,11 @@
     <refname>systemd-pcrphase-sysinit.service</refname>
     <refname>systemd-pcrphase-initrd.service</refname>
     <refname>systemd-pcrmachine.service</refname>
+    <refname>systemd-pcrproduct.service</refname>
     <refname>systemd-pcrfs-root.service</refname>
     <refname>systemd-pcrfs@.service</refname>
     <refname>systemd-pcrextend</refname>
-    <refpurpose>Measure boot phase into TPM2 PCR 11, machine ID and file system identity into PCR 15</refpurpose>
+    <refpurpose>Measure boot phases, machine ID, product UUID and file system identity into TPM PCRs and NvPCRs</refpurpose>
```

Comment:
UUIDs

Reply:
Maybe I misread, but the other bit says:

> measures the firmware product UUID (e.g. SMBIOS, Devictree, …) into...

So it says `UUID` singular, but then has multiple examples listed. So maybe it needs a bit of rephrasing to clarify?